services/blockchain-connector/shared/sdk/transactions.js

```diff
@@ -42,12 +42,14 @@ const getTransactionsFromPoolFormatted = async () => {
 };
 
 const dryRunTransactionWrapper = async (params) => {
-	const { transaction, skipVerify } = params;
+	const { transaction, skipVerify, skipDecode } = params;
 	const encodedTransaction = typeof transaction === 'object'
 		? encodeTransaction(transaction)
 		: transaction;
 
 	const response = await dryRunTransaction({ transaction: encodedTransaction, skipVerify });
+	if (skipDecode) return response;
```

Comment:
Better to give space for a control statement.

```suggestion

	if (skipDecode) return response;
```